common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -257,8 +266,48 @@ public BrokerAcquireTokenSilentOperationParameters brokerSilentParametersFromBun
         return new HashSet<>(Arrays.asList(scopes));
     }
 
+    public static Bundle getSilentBrokerRequestBundle(final AcquireTokenSilentOperationParameters parameters) {
```

Comment:
nit: lets try to follow the adapter conventions here, any reason why it's static?
also rename to `requestBundleFromSilentParameters` to be consistent and remove `static`

Reply:
After taking another look, it makes more sense to put these functions in BrokerBaseStrategy instead.

I'll move them there, and I'll remove static and rename the method for consistency also.